CHANGES.txt

```diff
@@ -1,4 +1,5 @@
 4.1
+ * Add Guardrail for GROUP BY queries (CASSANDRA-17509)
```

Comment:
Super nit: other similar entries in `CHANGES.txt` in use "Add guardrail...", without capitalizing.